settings.py

```diff
@@ -156,6 +156,7 @@
     MEDIA_URL = '/media/'
 
 LOADING_DATASETS_DIR = os.environ.get('LOADING_DATASETS_DIR')
+HAIL_SEARCH_DATA_DIR = os.environ.get('HAIL_SEARCH_DATA_DIR')
```

Comment:
👍 

Do we want to have defaults for these here or just have them as defaults in `sear-helm`?

Reply:
lets just have the defaulted in seqr helm

Reply:
👍